samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/KMLLayers.razor

```diff
@@ -0,0 +1,45 @@
+﻿@page "/kmllayers"
+
+<h3>Kml Layers</h3>
+
+<div class="links-div">
+    <a class="btn btn-secondary" target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-KMLLayer.html">ArcGIS API for JavaScript</a>
+</div>
+<Label>Add the sample KML Layer URL to see major earthquakes for latest 30 days from USGS to the map database :</Label>
+<InputText @bind-Value="_kmlLayerUrl"></InputText>
+<button disabled="@(!_mapRendered)" @onclick="(()=>AddKMLLayer())">Add new KML Layer!</button>
+
+
+<MapView @ref="_view" class="map-view" OnMapRendered="OnMapRendered" Scale="0">
+    <Map ArcGISDefaultBasemap="dark-gray-vector">
+        @if (_visible)
+        {
+            <KMLLayer Url="https://earthquake.usgs.gov/fdsnws/event/1/query?format=kml&minmagnitude=5.8" Title="Major EarthQuakes Query Result"></KMLLayer>
+        }
+    </Map>
+</MapView>
+
+
+
+@code {
+    private void OnMapRendered() => _mapRendered = true;
+
+    public async Task AddKMLLayer()
+    {
+        _visible = true;
+        await LoadKMlLayer();
```

Comment:
We're defining the KML layer twice, once in markup on line 17, and separately in C# on line 37. So you have added the same layer twice. I know @AndersenBell was suggesting that we try to show off the different ways of adding things, but I think this can be confusing and imply that you have to do _both_, and it would be more clear if they were separate layers, possibly triggered by separate buttons, like `Add Layer in Markup` and `Add Layer in Code`.